src/runtime/image.ts

```diff
@@ -106,6 +114,22 @@ async function getMeta (ctx: ImageCTX, input: string, options: ImageOptions) {
   return meta
 }
 
+async function fetchPayload (page, context) {
+  if (!context.nuxtState) {
```

Comment:
The page payload uses on client side in full static websites, And fetching payload on the server side is meaningless. We could simply check for existence of `window.__NUXT_IMPORT__` function and import payload

Reply:
should this be changed?